jib-gradle-plugin/README.md

```diff
@@ -212,12 +212,13 @@ Property | Type | Default | Description
 `entrypoint` | `List<String>` | *None* | The command to start the container with (similar to Docker's [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#entrypoint) instruction). If set, then `jvmFlags` and `mainClass` are ignored. You may also set `jib.container.entrypoint = 'INHERIT'` to indicate that the `entrypoint` and `args` should be inherited from the base image.\*
 `environment` | `Map<String, String>` | *None* | Key-value pairs for setting environment variables on the container (similar to Docker's [ENV](https://docs.docker.com/engine/reference/builder/#env) instruction).
 `extraClasspath` | `List<String>` | *None* | Additional paths in the container to prepend to the computed Java classpath.
+`filesModificationTime` | `String` | `EPOCH_PLUS_SECOND` | Sets the timestamps (last modified time) of files in the image put by Jib. (Note that this does not set the image creation time, which can be set using `<useCurrentTimestamp>`.) The value should be in ISO 8601 date time format parsable with [`DateTimeFormatter.ISO_DATE_TIME`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html). Examples are `2019-07-15T10:15:30+09:00`, `2011-12-03T22:42:05Z`, etc. You can also use the keyword `EPOCH_PLUS_SECOND` for the default behavior to set the timestamps to Epoch + 1 second.
```

Comment:
This read a bit weird:
> _The value should be in ISO 8601 [… lotsa text…]. You can also use the keyword …_

I kept thinking  "_but you use `EPOCH_PLUS_SECOND` as the default!_" as I read through the _lotsa text_.

```suggestion
`filesModificationTime` | `String` | `EPOCH_PLUS_SECOND` | Sets the timestamps (last modified time) of files in the image put by Jib. (Note that this does not set the image creation time, which can be set using `<useCurrentTimestamp>`.) The value should either be `EPOCH_PLUS_SECOND` to set the timestamps to Epoch + 1 second (default behavior), or an ISO 8601 date time parsable with [`DateTimeFormatter.ISO_DATE_TIME`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html) such as `2019-07-15T10:15:30+09:00` or `2011-12-03T22:42:05Z`.
```

Reply:
That's a lot clearer. Thanks for the convenient suggestions.